admin/spec/service/pin-generation.service.spec.js

```diff
@@ -190,6 +190,7 @@ describe('pin-generation.service', () => {
         const result = pinGenerationService.generateSchoolPassword(school)
         expect(result.pinExpiresAt).toBeDefined()
         expect(result.schoolPin.length).toBe(8)
+        expect(/[a-z]{3}[2-9]{2}[a-z]{3}/.test(result.schoolPin)).toBe(true)
```

Comment:
I think this would benefit from anchors: `/^[a-z]{3}[2-9]{2}[a-z]{3}$/ otherwise it would match nonsense like `222abc99def111`